### ISSUE TITLE
*hax payload title takeover fails with sploit_installer on >=v11.2

### ISSUE THREAD

**Issue by RapTix** [yellows8/stickerhax#6]
I'm trying to install stickerhax, but when I select sploit installer stickerhax in the homebrew menu my screen turns yellow and my 3ds freezes.

**Comment by yellows8** [comment]
No, it's just *hax-payload title-takeover with sploit_installer. https://github.com/smealum/ninjhax2.x

**Comment by RapTix** [comment]
I am following this video step by step https://www.youtube.com/watch?v=eMV1NIKiY4c (well its for an older fm version but it should work that way shouldn't it?) But befor I get into the sploit installer screen (2:06 in the video) my screen turns yellow and nothing happens at all. Sorry for my bad english.
What am I doing wrong?

**Comment by yellows8** [comment]
Which is exactly what I meant... "What am I doing wrong?" Nothing.

**Comment by alrulu** [comment]
I have the same problem

**Comment by alrulu** [comment]
It happens to me on steelhax too

**Comment by alrulu** [comment]
are you working on this?

**Comment by alrulu** [comment]
Happens to me with all the haxs, what happens? I do something wrong?
